Directory.Build.props

```diff
@@ -279,6 +279,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <!-- By default the SDK produces ref assembly for 5.0 or later -->
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+    <!-- Since we don't use implicit references in most of our projects we need to disable implicit namespace imports
+    to avoid build failures about missing namespaces. -->
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```

Comment:
Can https://github.com/dotnet/runtime/blob/978b0dbc5e1475be917f4237edf5b47f4c124dba/src/coreclr/tools/Directory.Build.props#L7 be removed?

Reply:
Now that I moved it to where we only need it, we should leave this one as is.